src/components/publishing/image.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react"
+import styled from "styled-components"
+import Fonts from "./fonts"
+
+const Figcaption = styled.div`
+  & > p {
+    ${Fonts.unica("s14")}
+    margin: 0;
+    color: #999;
+  }
+`
+
+const CaptionLink = styled.div`
+  ${Fonts.unica("s14")}
+  margin: 0;
+  border-bottom: 1px solid black;
+  cursor: pointer;
+  display: inline-block;
```

Comment:
might be a good idea to add padding left here, so that long captions don't sit too close